docs/plugins/backup.md

```diff
@@ -46,7 +46,7 @@ plugin(
     b.bucket = 'acquia-cloud-database-test'
     b.files = [
```

Comment:
Fixed. I'm not really sure what this has to do with how filesystems work, although it's true that I wasn't aware of the existence of that function.